docs/architecture/adr-061-liquid-staking.md

```diff
@@ -0,0 +1,81 @@
+
+As both Proof of Stake and blockchain use cases have matured, this design has aged poorly and should no longer be considered a good baseline Proof of Stake issuance. In the world of application specific blockchains, there cannot be a one size fits all blockchain but the Cosmos SDK does endeavour to provide a good baseline implementation and one that is suitable for the Cosmos Hub.
+
+The most important deficency of the legacy staking design is that it composes poorly with on chain protocols for trading, lending, derivatives that are referred to collectively as DeFi. The legacy staking implementation starves these applications of liquidity by increasing the risk free rate adaptively. It basically makes DeFi and staking security somewhat incompatible. 
+
+The Osmosis team has adopted the idea of Superfluid and Interfluid staking where assets that are participating in DeFi appliactions can also be used in proof of stake. This requires tight integration with an enshrined set of DeFi applications and thus is unsuitable for the Cosmos SDK.
+
+It's also important to note that Interchain Accounts are available in the default IBC implementation and can be used for staking and thus reyhypothneciation and liquid staking of staked assets is already possible. Thus liquid staking is already possible and these changes improve the UX of liquid staking. Centralized exahnges have also provided rehypthentication of staked assets and posed a challenge for decentralization. This ADR also take the position that liquid staking adoption is good and provides new levers to incentivize decentralization of stake. The
+
+These changes to the staking module have been in development for more than a year and have seen substantial industry adoption who plan to build staking UX. The internal economics at Informal team has also done a review of the impacts of these changes and this review led to the developement of the exempt delegation system. This system provides governance with a tunenable parameter for modulating the risks of pricipal agent problem called the exemption factor. 
+
+## Decision
+
+We implement the semi-fungible liquid staking system and exemption factor system within the cosmos sdk.
+
+A new governance parameter is introduced that defines the ratio of exempt to issued tokenized shares. This is called the exemption factor.
+
+Min self delegation is removed from the staking system with the expectation that it will be replaced by the exempt delegations sytem.
+
+When shares are tokenized, the underlying shares are transfered to a module account and rewards go to the module account for the TokenizedShareRecord. 
+
+There is no longer a mechanism to override the validators vote for TokenizedShares.
```

Comment:
I am curious to understand,
Why liquid staking voting that overrides validators' vote is not included which helps validator decentralization for the chain?

Reply:
Once the the economic stake has been made transferable, it's unclear whom should hold the vote rights. 

IF you want to recover the override capabiltiy, simply redeem the tokenized shares. This can be done via an interchain account for some liquid staking designs.